<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>240</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closseButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelState">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GAME STATE</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="nextTetrominoView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>110</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNextTetromino">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Next tetromino:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="holdTetrominoView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>110</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHold">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelScore">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GAME SCORE</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTetrisCount">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TETRIS COUNT</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLevel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LEVEL</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
